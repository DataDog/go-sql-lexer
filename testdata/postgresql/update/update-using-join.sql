UPDATE orders SET total = total * 0.9 FROM users WHERE users.id = orders.user_id AND users.status = 'VIP';
