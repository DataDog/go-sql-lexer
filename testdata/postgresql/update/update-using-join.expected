[
    {
        "expected": "UPDATE orders SET total = total * ? FROM users WHERE users.id = orders.user_id AND users.status = ?",
        "statement_metadata": {
            "size": 17,
            "tables": [
                "orders",
                "users"
            ],
            "commands": [
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]